lib/internal/http2/core.js

```diff
@@ -844,7 +778,7 @@ class Http2Session extends EventEmitter {
 
     const handle = this[kHandle];
     if (handle === undefined)
-      return Object.create(null);
+      return {};
```

Comment:
Q: Why is this preferred? (IIRC we discussed this before so I'm curious why change it now.)

Reply:
Under turbofan using the literal is more performant it turns out, and after evaluating, there's no real benefit to making these particular objects have that null prototype

Reply:
IIRC, since about the time of TF being enabled (not sure it’s related), `Object.create(null)` acts as a hint to the engine that the object is in dictionary mode, i.e. kind of acts like a Map.

So not using a null-prototype object here seems like a good idea.

Reply:
TIL 🌈✨ Thanks!